aiida_testing/mock_code/_cli.py

```diff
@@ -27,23 +28,44 @@ def run() -> None:
     directory.
     """
     # Get environment variables
+    log_path = Path(os.environ[EnvKeys.LOG_FILE.value])
     label = os.environ[EnvKeys.LABEL.value]
     data_dir = os.environ[EnvKeys.DATA_DIR.value]
     executable_path = os.environ[EnvKeys.EXECUTABLE_PATH.value]
     ignore_files = os.environ[EnvKeys.IGNORE_FILES.value].split(':')
     ignore_paths = os.environ[EnvKeys.IGNORE_PATHS.value].split(':')
     regenerate_data = os.environ[EnvKeys.REGENERATE_DATA.value] == 'True'
+    fail_on_missing = os.environ[EnvKeys.FAIL_ON_MISSING.value] == 'True'
 
-    hash_digest = get_hash().hexdigest()
+    def _log(msg: str) -> None:
+        """Write a message to the log file."""
+        with open(log_path, 'a', encoding='utf8') as log_file:
```

Comment:
I think this should be ok, for potentially concurrent writes to the log file?
Or do you write a separate log file, for every mock code execution?

Reply:
I think this solution is fine.
It should indeed allow concurrent writes, and in the end this is about debugging; people will be able to read this info - let's not make it more complicated then needed